tests/test_parallelization.py

```diff
@@ -74,3 +74,25 @@ def test_parallel_subcommand_relative_ordering() -> None:
     assert thrd_sc["angles"] == [[0.5, 0.0], "pi"]
     assert frth_sc["qop"] == "RZ"
     assert frth_sc["angles"] == [[3.5], "pi"]
+
+
+def test_single_qubit_circuit_with_parallel() -> None:
+    """Make sure there are no parallel blocks present in the 1qubit circuit."""
+    phir_parallel = get_phir_json(QasmFile.single_qubit_parallel_test, rebase=True)
+    phir_no_parallel = get_phir_json(QasmFile.single_qubit_parallel_test, rebase=False)
```

Comment:
@Asa-Kosto-QTM I did have a question while staring at this test -- aren't both of these no parallel? If so, perhaps the names should be `rebased` and `no_rebased`.

Reply:
with no rebasing, there is no parallelization, when there is a machine object, there is parallelization. I think the rebasing is a good proxy but I can call have it get the json by calling the actual phirgen functions if you think it would be more explicit

Reply:
I was only referring to the naming of those variables; it's misleading to distinguish these two with `*_parallel` and `*_no_parallel` when there is no parallelism expected in either.

Another potential improvement to the test is to ensure there is no `"qparallel"` block in the first phir output. However, this seems implicit as you compare `"args"` with the non-rebased phir output.